dependencies64/ffmpeg/ffmpeg-build-linux/build-ubuntu.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
+#compile ffmpeg successful on ubuntu 14.04.3 & 16.04.1 amd64 using the CasparCG build requirments described here:
+#https://raw.githubusercontent.com/casparcg/Server/2.1.0/BUILDING
 
-sudo apt-get install build-essential curl tar pkg-config
+sudo apt-get -y update
+sudo apt-get -y upgrade
```

Comment:
It is common practice that you make the changes. These comments on lines of code is often used as a code review tool, meaning that the merge will not be done until the problems have been fixed in the pull. I haven't tried the "Start a review" button, maybe it offers are more formal way of handling it.